eq-author-api/schema/typeDefs.js

```diff
@@ -577,7 +577,7 @@ type Mutation {
   updateBinaryExpression2(input: UpdateBinaryExpression2Input!): BinaryExpression2!
   updateLeftSide2(input: UpdateLeftSide2Input!): BinaryExpression2!
   updateRightSide2(input: UpdateRightSide2Input!): BinaryExpression2!  
-  deleteBinaryExpression2(input: DeleteBinaryExpression2Input!): BinaryExpression2!
+  deleteBinaryExpression2(input: DeleteBinaryExpression2Input!): ExpressionGroup2!
```

Comment:
How come we need to return the parent here?

Reply:
Due to not storing a back-ref to the parent the only way of determining the correct expression group is to look for the one that contains the expression. Since the expression is deleted you cannot resolve the parent.

It's only an issue when you delete something and want to get back to the parent from the returned deleted entity. In this case it is simpler to just return the parent directly.

Possibly there might be a nicer way of doing this.